lib/crypto.js

```diff
@@ -633,6 +633,37 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
+function randomFillSync(buf, offset = 0, size) {
+  if (!Buffer.isBuffer(buf)) {
+    throw new TypeError('"buf" argument must be a Buffer');
+  }
+
+  if (!size) size = buf.length - offset;
+
+  return binding.randomFill(buf, offset, size);
+}
+exports.randomFillSync = randomFillSync;
```

Comment:
Just running through a few argument examples on this, most variations look good with one exception:

```js
>crypto.randomFillSync(Buffer.alloc(10), 11)
TypeError: size must be a number >= 0
```
The issue, of course, is in how the default size is calculated if the argument is not given but the error message is confusing in this case because it does not match the actual problem -- the fact that the offset is greater than the buffer size.